Assets/_BForBoss/_Entities/Scripts/LifeCycleBehaviour.cs

```diff
@@ -67,7 +67,7 @@ protected virtual void LifeCycleDamageTaken()
             }
         }
 
-        protected virtual void CleanUp()
+        public virtual void CleanUp()
```

Comment:
Nitpick: public should be above protected

Reply:
Fixed